sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -75,26 +75,203 @@
  * credentials for accessing the storage or CosmosDB table API account. It provides methods to create and delete the
  * table itself, as well as methods to create, upsert, update, delete, list, and get entities within the table. These
  * methods invoke REST API operations to make the requests and obtain the results that are returned.</p>
  *
- * <p>Instances of this client are obtained by calling the {@link TableClientBuilder#buildAsyncClient()} method on a
- * {@link TableClientBuilder} object.</p>
+ * <h2>Getting Started</h2>
+ *
+ * <p>Authenticating and building instances of this client are handled by {@link TableClientBuilder}.
+ * This sample shows how to authenticate and build a TableClient instance using the {@link TableClientBuilder} and
+ * a connection string.</p>
  *
- * <p><strong>Samples to construct an async client</strong></p>
- * <!-- src_embed com.azure.data.tables.tableAsyncClient.instantiation -->
+ * <!-- src_embed com.azure.data.tables.tableAsyncClient.instantiation.connectionstring -->
  * <pre>
  * TableAsyncClient tableAsyncClient = new TableClientBuilder&#40;&#41;
- *     .endpoint&#40;&quot;https:&#47;&#47;myaccount.core.windows.net&#47;&quot;&#41;
- *     .credential&#40;new AzureNamedKeyCredential&#40;&quot;name&quot;, &quot;key&quot;&#41;&#41;
+ *     .connectionString&#40;&quot;connectionstring&quot;&#41;
  *     .tableName&#40;&quot;myTable&quot;&#41;
  *     .buildAsyncClient&#40;&#41;;
  * </pre>
- * <!-- end com.azure.data.tables.tableAsyncClient.instantiation -->
+ * <!-- end com.azure.data.tables.tableAsyncClient.instantiation.connectionstring -->
+ *
+ * <p>For more information on building and authenticating, see the {@link TableClientBuilder} documentation.</p>
+ *
+ * <p>The following code samples provide examples of common operations preformed with this client.</p>
+ *
+ * <hr/>
+ *
+ * <h3>Create a {@link TableEntity}</h3>
+ *
+ * <p>The {@link #createEntity(TableEntity) createEntity} method can be used to create a table entity within a table in your Azure Storage or Azure Cosmos account.</p>
+ *
+ * <p>The sample below creates a {@link TableEntity} with a partition key of "partitionKey" and a row key of "rowKey".</p>
+ *
+ * <!-- src_embed com.azure.data.tables.tableAsyncClient.createEntity#TableEntity -->
+ * <pre>
+ * TableEntity tableEntity = new TableEntity&#40;&quot;partitionKey&quot;, &quot;rowKey&quot;&#41;
+ *     .addProperty&#40;&quot;Property&quot;, &quot;Value&quot;&#41;;
+ *
+ * tableAsyncClient.createEntity&#40;tableEntity&#41;
+ *     .contextWrite&#40;Context.of&#40;&quot;key1&quot;, &quot;value1&quot;, &quot;key2&quot;, &quot;value2&quot;&#41;&#41;
+ *     .subscribe&#40;unused -&gt;
+ *         System.out.printf&#40;&quot;Table entity with partition key '%s' and row key '%s' was created.&quot;, &quot;partitionKey&quot;,
+ *             &quot;rowKey&quot;&#41;&#41;;
+ * </pre>
+ * <!-- end com.azure.data.tables.tableAsyncClient.createEntity#TableEntity -->
+ *
```

Comment:
Add here and for other samples,
 * <p><strong>Note:</strong> for asynchronous sample, refer to {@link async client}</p>